DOCS-3692-made edits (#169)
mulesoft/docs-api-manager: modules/ROOT/pages/policy-template-source.adoc

> endif::[]

Reviewer: I don't understand the title of this topic. 
Is this a Level-1 topic about a URL? 
That sounds pretty narrow for a major topic.

Reviewer: GLOBALLY replace the possessive ('s) with just the plain term, as an adjective: Policy Template Source URL (although I don't think that this is the title we want here...).

Reviewer: This current title implies that the following text is about the URL from which to source policy templates.
I don't think that this is what you intend, that the topic be simple a URL.
What is the user task, here, and WHY does your audience want or need the information in this topic?

Reviewer: I suggest that you change the topic title to something like "Download Policy Templates" or "Obtain the URL to Download Policy Templates."
Then I think the text should begin by explaining the circumstances in which your audience wants or needs to do this or to have this information: where is the user in all of this?

:keywords: policy, CloudFront

> The configuration parameters described in this section are relevant to only Mule runtime engine (Mule) versions 4.1.7 and later.

Reviewer: The topic should not begin with this text. The audience has no reference for "the parameters" yet.
The topic should begin by introducing what is covered here and WHY the audience wants or needs it.

> When an application with a tracked flow is deployed to a Mule environment, the API gateway of that Mule environment communicates with API Manager to request for the applied policies.

Reviewer: Where is the user/audience? Is this accurate:
```suggestion
When you deploy an application with a tracked flow to a Mule environment, the API gateway of that Mule environment communicates with API Manager to request the policies applied to the environment.
```
Does the application require the policies or the policy templates for the environment? The next sentence refers to the latter; this sentence the former.

> If the templates for the policies that are to be applied are not already present in Mule, API gateway requests API Manager for the URLs from where to download the template.

Reviewer: The previous sentence refers to applied policies; this one refers to templates for policies that are to be applied (not policies: templates, and not applied: to be applied). The two sentences are inconsistent, right at the start of the topic.

Are the policies applied or are there just templates for the policies to be applied, templates that the user has to configure? Again, I have no idea what the user is supposed to DO here, or why.

> You can configure the following property to provide Policy templates to API Manager from a specified source:

Reviewer: Is this something that you do before or after the situation introduced in the previous two sentences?

Reviewer: This sentence says that you configure a property that obtains "from a specified source" a policy template that is then given TO API Manager...when the previous text states that the gateway attempts to get the policy template FROM API Manager. The flow for obtaining the template is unclear. What is the "specified source"? Is that the URL? Where does the information about the URL travel from, and where end?

`anypoint.platform.policy_template_source`

> [%header,cols="2*a"]

Reviewer: The syntax above does not include the parameters "external" and "internal" that follow in the table.
How are the syntax and the table related?

Reviewer: Note that we should not be using a table to provide a term/syntax and its definition; we should use a definition or description list, per our style.

Reviewer: If these are the "parameters" referred to at the beginning, then this is the appropriate place to mention versioning restrictions.

|===
|Value | Description

> |external | Policy templates to be retrieved from the legacy system. This is the default value to maintain compatibility with previously configured firewalls.

Reviewer: Does your audience know what the legacy system is, and why it is external?
Do we really need to mention the firewall? Isn't this irrelevant when the alternative is internal sourcing?

Reviewer: ```suggestion
|external | Policy templates are retrieved from the external legacy system. This is the default value to maintain compatibility with previously configured firewalls.
```

> |internal | Policy templates to be retrieved from the new system. This option provides additional reliability due to internal rerouting of the requests when a server becomes unavailable.

Reviewer: Are the templates actually "retrieved" from the new system or are they just there "within" the new system?
```suggestion
|internal | Policy templates are retrieved from within the new system. This option provides additional reliability due to internal rerouting of requests when a server becomes unavailable.
```

|===

> If you start Mule version 4.2.2 , as shown in the example, API Manager provides the URLs for downloading the policy templates from the Mule CloudFront service.

Reviewer: Does the example really show how the syntax configuration defined above works in practice?

Reviewer: Suggested rewording:
```suggestion
The following example shows a Mule 4.2.2 environment in which API Manager provides the URLs for downloading policy templates from the Mule CloudFront service:
```

Reviewer: Why is Mule CloudFront service not mentioned in the topic intro? Is it always the template source?
